src/main/java/jalau/at18/katas/likes/mauricio/Main.java

```diff
@@ -0,0 +1,50 @@
+package jalau.at18.katas.likes.mauricio;
+
+public class Main {
+    private static final int FIRST_CASE = 0;
+    private static final int SECOND_CASE = 1;
+    private static final int THIRD_CASE = 2;
+    private static final int FOURTH_CASE = 3;
+    private static final int FIFTH_CASE = 4;
+
+    public static void main(String[] args) {
+        String[] nameList = {"Carlos", "Alberto", "Karina", "Liz"};
+        String likes = getLikesFromPost(nameList);
+        System.out.println(likes);
+    }
+
+    static String getLikesFromPost(String[] nameList) {
+        String first_name = "None";
```

Comment:
You are using snake_case for you variables, which is ok, but I'd recommend to use camelCase so it keeps consistency for all the project.

For constants, we can keep SNAKE_CASE